internals/secrethub/run.go

```diff
@@ -99,15 +102,17 @@ func (cmd *RunCommand) Run() error {
 		cmd.command = strings.Split(cmd.command[0], " ")
 	}
 
-	valuesToMask := make([][]byte, 0, len(secrets))
+	sequences := make([]*regexp.Regexp, 0, len(secrets))
 	for _, val := range secrets {
 		if val != "" {
-			valuesToMask = append(valuesToMask, []byte(val))
+			sequences = append(sequences, regexp.MustCompile(regexp.QuoteMeta(val)))
 		}
 	}
 
-	maskedStdout := masker.NewMaskedWriter(cmd.io.Stdout(), valuesToMask, maskString, cmd.maskingTimeout)
-	maskedStderr := masker.NewMaskedWriter(os.Stderr, valuesToMask, maskString, cmd.maskingTimeout)
+	m := masker.Masker{
+		BufferDelay:    time.Millisecond * 100,
+		MatchSequences: sequences,
+	}
```

Comment:
Shall we create these regex sequences in a Masker constructor? No need to expose that we're using regexes internally to match the sequences.

Reply:
That makes it easier to use and for us to change it later (e.g. to improve performance).